docs/GettingStarted.md

```diff
@@ -42,6 +42,10 @@ This section outlines the steps to build, prepare and build the eBPF-For-Windows
 1. ```git clone --recurse-submodules https://github.com/microsoft/ebpf-for-windows.git```.
 By default this will clone the project under the `ebpf-for-windows` directory.
 
+#### Exclusion of PE parse directory from Windows Security
```

Comment:
I would call it Windows Defender Antivirus. Also you must add a line that similar exclusions have to be done for other Antivirus products as needed. You could also look at the slack channel response that Alan made for additional context as to why this workaround is needed.

Reply:
Good point. I added some details from the slack channel and a note for other antiviruses as well.